CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Allow disabling service links (environment variables describing Kubernetes services) in Pod template
 * Update Kaniko executor to 1.6.0
 * Add support for separate control plane listener (disabled by default, available via the `ControlPlaneListener` feature gate)
+* Add support for filtering logs by the markers
```

Comment:
Does everyone understand what does it mean and what is it good for?

Reply:
```suggestion
* Add support for selectively changing the verbosity of logging for individual CRs, using markers.
```

Reply:
That sounds good to me 👍